rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/model/RedrivePolicyJson.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * All right reserved - Fronteer LTD
```

Comment:
I'm not sure that applies when contributing to an open-source project? (maybe it doesn't, but I'm not very familiar with copyright law :) )

Reply:
Oh dear, I must have a global template for new files. I'll take this out, good catch!